### PR TITLE
BLT-2072: temporarily disabling checking of features overrides.

### LINKED ISSUE #2072: BLT / Features Incompatibility
My system information:
* Operating system type: Mac OS
* Operating system version: 10.11.6
* BLT version: 8.x-dev cfe62bd53f800e5c2dcb39045fbf23a24b99192a

When I run this command:
```
blt setup
```

I get the following output:
```
Checking for features overrides...
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/<project>/vendor/bin/drush fl --bundle=<project> --uri=default in /var/www/<project>/docroot
PHP Fatal error:  Call to undefined function drush_hide_output_fields() in /var/www/<project>/docroot/modules/contrib/features/drush/features.drush.inc on line 218

Fatal error: Call to undefined function drush_hide_output_fields() in /var/www/<project>/docroot/modules/contrib/features/drush/features.drush.inc on line 218
 [error]  Drush command terminated abnormally due to an unrecoverable error.
Error: Call to undefined function drush_hide_output_fields() in /var/www/<project>/docroot/modules/contrib/features/drush/features.drush.inc, line 218 
[Acquia\Blt\Robo\Tasks\DrushTask]  Exit code 255  Time 3.967s
[error]  Unable to determine if features in bundle <project> are overridden. 
[error]  Command `setup:config-import ` exited with code 1. 
Connection to 127.0.0.1 closed.
macbookpro-mikemadison:<project> mike.madison$ drush @<project>.local features-list-packages --format=json --bundle=<project>
 [error]  Unknown option: --format.  See `drush help features-list-packages` for available options. To suppress this error, add the option --strict=0. 
Connection to local.<project>.com closed.
 [error]  Unknown option: --format.  See `drush help features-list-packages` for available options. To suppress this error, add the option --strict=0. 
```

And I expected this to happen:

It would appear that Features is not compatible with Drush 9. I have bugged this issue here: https://www.drupal.org/node/2913651

In the meantime, the thing that is exploding is the final step where we check for features overrides because the features-list-packages code has a function that isn't supported in Drush 9. We could potentially remove this portion of the features import process and bypass the incompatibility.

### PR DESCRIPTION
**Pull request by mikemadison13**
Fixes #2072  .

Changes proposed:
- comment out the check features override function to prevent potential fatal errors due to Drush 9 incompatibility

### CODE REVIEW

File: src/Robo/Commands/Setup/ConfigCommand.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ public function import() {
         throw new BltException("Failed to import configuration!");
       }
 
-      $this->checkFeaturesOverrides();
+      //$this->checkFeaturesOverrides();
```
Comment (grasmash) [inline src/Robo/Commands/Setup/ConfigCommand.php]: Can you do this only for Drush 9?

Comment (grasmash) [reply src/Robo/Commands/Setup/ConfigCommand.php]: `$this->getInspector()->getDrushMajorVersion() == 8`

**Comment by danepowell** [comment]
> PHP Fatal error:  Uncaught Error: Call to undefined method Acquia\Blt\Robo\Inspector\Inspector::getDrushMajorVersion() in /home/travis/build/acquia/blt/src/Robo/Commands/Setup/ConfigCommand.php:88

**Comment by grasmash** [comment]
It will be merged in #2080.

**Comment by grasmash** [comment]
#2082

**Pull request closed without merging**